fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/DescribeCommand.java

```diff
@@ -270,11 +274,11 @@ public ShowResultSet doRun(ConnectContext ctx, StmtExecutor executor) throws Exc
                     builder.deleteCharAt(builder.length() - 1);
                     procString += builder.toString();
                 }
-                node = ProcService.getInstance().open(procString);
+                ProcNodeInterface node = ProcService.getInstance().open(procString);
                 if (node == null) {
                     throw new AnalysisException("Describe table[" + dbTableName.getTbl() + "] failed");
                 }
-                rows.addAll(getResultRows());
+                rows.addAll(getResultRows(node));
             } else {
                 Util.prohibitExternalCatalog(dbTableName.getCtl(), this.getClass().getSimpleName() + " ALL");
```

Comment:
how about desc table all command? should we show comment column based on showColumnCommentInDescribe session var?

Reply:
I though about that, but most of use only know `desc table` statement. And their BI tools relies on this syntax. If we support it in `desc table all`, their have to change their business code.